src/main/java/frc/robot/commands/intakeCube.java

```diff
@@ -28,10 +28,10 @@ public intakeCube(Arm subArm, Collector subCollector, Intake subIntake) {
         new InstantCommand(() -> subCollector.setPivotMotorAngle(prefCollector.pivotAngleCubeCollecting.getValue())),
 
         // - Move arm to collector
-        new InstantCommand(
-            () -> subArm.setJointPositions(
-                prefArm.armPresetCollectorElbowAngle,
-                prefArm.armPresetCollectorShoulderAngle)),
+        // new InstantCommand(
```

Comment:
Shouldn't we replace the setJointPositions to setGoalAngles instead of commenting it out here?

Reply:
yes, but thats a functional change of another class. `setJointPositions()` is nonfunctional when used outside of the MoveArm command, so commenting it out of `intakeCube` will just make the actual behavior more clear